WhaleSpotting/react-app/src/components/ReportSighting.tsx

```diff
@@ -32,11 +33,15 @@ export default function ReportSighting(): JSX.Element {
         };
 
         let isError = false;
-        const response = await fetch("api/sightings/create", {
+
+        const token = await authService.getAccessToken();
```

Comment:
Can we make this a method in the api client please. We can rename the getHeaders method to getGetSettings and add a getPostSettings so that we can just use a `const response = await fetch("sightings/create", await getPostSettings(sighting));`